src/bindings/c/ov/tests/ov_c_api_test.cpp

```diff
@@ -0,0 +1,1718 @@
+// Copyright (C) 2018-2022 Intel Corporation
```

Comment:
Can we split this huge file to several files which test different features?

Reply:
We follow the legacy C API, do we need split them?